sysconfig/marconi/modules.yaml

```diff
@@ -0,0 +1,39 @@
+modules:
+  enable::
```

Comment:
there is double :

Reply:
double colon is to override parent configs (i.e. if there is modules block in default configs, we just ignore them)